src/main/java/org/owasp/html/Encoding.java

```diff
@@ -37,7 +37,7 @@
 
   /**
    * Decodes HTML entities to produce a string containing only valid
-   * Unicode scalar values.
+   * Unicode code points.
```

Comment:
Updated since this works on surrogate pairs which are not scalar values.

Reply:
The difference between scalar values and code points are orphaned surrogates so I don't see what surrogate pairs have to do with this.

Reply:
:+1: I misunderstood. Will revert.

Reply:
Reverted